src/components/FrontPage/Filter.js

```diff
@@ -44,4 +69,19 @@ const Filter = React.createClass({
   }
 });
 
-export default Filter;
+Filter.propTypes = {
+  onFilterCheck: PropTypes.func,
+};
+
+function mapStateToProps(state) {
```

Comment:
Ingenting galt her, men for å gjøre det littegrann mer leselig, kan jeg foreslå følgende?
```
const mapStateToProps = (state) => ({
  filter: state.filter,
});

const mapDispatchToProps = {
  onFilterCheck
};

export default connect(
  mapStateToProps,
  mapDispatchToProps,
)(Filter);
```